includes/admin/importers/class-wc-tax-rate-importer.php

```diff
@@ -166,7 +166,7 @@ public function import( $file ) {
 		}
 
 		// Show Result
-		echo '<div class="updated settings-error below-h2"><p>
```

Comment:
Since this importer page show display notice after h1, its ok we remove `below-h2` class :)